### DISCUSSION TITLE
A bot to acknowledge contributors

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [devicons/devicon#466] [Ideas]
Hi all,

I just found this pretty cool [bot](https://allcontributors.org/) that would acknowledge contributors to our repo. I was thinking that we can add the bot to our repo to show appreciation to our contributors in the `README.md`. Devicon wouldn't be what it is now without this great community's time and efforts.

What do you guys think?

**Comment by amacado** [comment]
I like the idea of giving credit to all contributors, regardless of what they are contributing. But in my opinion this just adds a functionality which is already done by github itself: https://github.com/devicons/devicon/graphs/contributors

![image](https://user-images.githubusercontent.com/8781699/103795151-ecc17900-5045-11eb-84e9-6af075253419.png)

While https://allcontributors.org/ is surely more handsome, but it also requires someone to always think about adding a new contributor to the list (even if it's just a comment - someone has to remember it). In addition of merging all those pull requests created by the bot every time a contributor is added.. In my opinion it's not worth the effort. But this is just my opinion and anyone is allowed to disagree.

**Comment by Thomas-Boi** [comment]
I was under the impression that normal users can't see the 'Insights' section of the repo. If GitHub can provides and anyone can access it, I think we probably don't need the bot then. It can still be a cool side project but I think the current system should be fine.

**Reply by Snailedlt** [reply]
Now we can see it in the bottom right corner of the repo: https://github.com/devicons/devicon

But, if it's mirrored elsewhere, or when viewing locally you won't see that. So we could add it to the readme like you suggest, but I agree with what @amacado said, that it would be too much effort for a very small thing.

Enter [contrib.rocks](https://contrib.rocks). Contrib.rocks let's you generate a contributors image based on the repo name. So we could simply go there whenever we make a new release, and generate a new image. Here's how it looks currently, using the following link to generate: https://contrib.rocks/preview?repo=devicons%2Fdevicon

<a href="https://github.com/devicons/devicon/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=devicons/devicon" />
</a>

**Reply by Snailedlt** [reply]
There's also these options to limit the number of contributors shown:
>Available options (Add to image URL query params)
max={number}
Max contributor count (100 by default)
columns={number}
Max columns (12 by default)

**Reply by amacado** [reply]
With this solution there would be no need for a bot or manual update on release.. You could just paste the dynamic image generation url in the readme, like this:

```
<a href="https://github.com/devicons/devicon/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=devicons/devicon" />
</a>
```

<a href="https://github.com/devicons/devicon/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=devicons/devicon" />
</a>

**Reply by Snailedlt** [reply]
Exactly!
The disadvantage is that the icons aren't clickable, and you also can't show any extra information other than their image.

With the all-contributors bot, you can automate adding people to the list, and you can give them different "roles" or "icons" depending on what they've contributed with.
example from all-contributors official repo: https://github.com/all-contributors/all-contributors#contributors-

I think if we have time to implement all-contributors in a mostly automated fashion, it would be pretty cool to see exactly what people contributed with

**Reply by Snailedlt** [reply]
Here's the example btw:

<table>
  <tr>
    <td align="center"><a href="https://kentcdodds.com"><img src="https://avatars.githubusercontent.com/u/1500684?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Kent C. Dodds</b></sub></a><br /><a href="#question-kentcdodds" title="Answering Questions">💬</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=kentcdodds" title="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Akentcdodds" title="Reviewed Pull Requests">👀</a> <a href="#talk-kentcdodds" title="Talks">📢</a></td>
    <td align="center"><a href="https://github.com/jfmengels"><img src="https://avatars.githubusercontent.com/u/3869412?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Jeroen Engels</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=jfmengels" title="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Ajfmengels" title="Reviewed Pull Requests">👀</a> <a href="#tool-jfmengels" title="Tools">🔧</a></td>
    <td align="center"><a href="https://jakebolam.com"><img src="https://avatars2.githubusercontent.com/u/3534236?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jake Bolam</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=jakebolam" title="Documentation">📖</a> <a href="#tool-jakebolam" title="Tools">🔧</a> <a href="#infra-jakebolam" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#maintenance-jakebolam" title="Maintenance">🚧</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Ajakebolam" title="Reviewed Pull Requests">👀</a> <a href="#question-jakebolam" title="Answering Questions">💬</a></td>
    <td align="center"><a href="https://github.com/tbenning"><img src="https://avatars2.githubusercontent.com/u/7265547?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Tyler Benning</b></sub></a><br /><a href="#maintenance-tbenning" title="Maintenance">🚧</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=tbenning" title="Code">💻</a> <a href="#design-tbenning" title="Design">🎨</a></td>
    <td align="center"><a href="https://sinchang.me"><img src="https://avatars0.githubusercontent.com/u/3297859?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jeff Wen</b></sub></a><br /><a href="#maintenance-sinchang" title="Maintenance">🚧</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Asinchang" title="Reviewed Pull Requests">👀</a></td>
    <td align="center"><a href="http://maxcubing.wordpress.com"><img src="https://avatars0.githubusercontent.com/u/8260834?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Maximilian Berkmann</b></sub></a><br /><a href="#translation-Berkmann18" title="Translation">🌍</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=Berkmann18" title="Documentation">📖</a> <a href="#maintenance-Berkmann18" title="Maintenance">🚧</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3ABerkmann18" title="Reviewed Pull Requests">👀</a> <a href="#talk-Berkmann18" title="Talks">📢</a></td>
    <td align="center"><a href="http://matheu.srv.br"><img src="https://avatars0.githubusercontent.com/u/23284276?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Matheus Rocha Vieira</b></sub></a><br /><a href="#translation-MatheusRV" title="Translation">🌍</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=MatheusRV" title="Code">💻</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=MatheusRV" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://robertlluberes.com"><img src="https://avatars1.githubusercontent.com/u/13991439?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Robert Lluberes</b></sub></a><br /><a href="#translation-robertlluberes" title="Translation">🌍</a></td>
    <td align="center"><a href="https://jongjineee.github.io"><img src="https://avatars2.githubusercontent.com/u/26620470?v=4?s=100" width="100px;" alt=""/><br /><sub><b>이종진</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=Jongjineee" title="Documentation">📖</a> <a href="#translation-Jongjineee" title="Translation">🌍</a></td>
    <td align="center"><a href="http://marsx.vip"><img src="https://avatars2.githubusercontent.com/u/21303543?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Wenqing Xue</b></sub></a><br /><a href="#translation-MarsXue" title="Translation">🌍</a></td>
    <td align="center"><a href="http://bogas04.github.io"><img src="https://avatars.githubusercontent.com/u/6177621?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Divjot Singh</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=bogas04" title="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Abogas04" title="Reviewed Pull Requests">👀</a></td>
    <td align="center"><a href="http://beneb.info"><img src="https://avatars.githubusercontent.com/u/1282980?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Ben Briggs</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=ben-eb" title="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Aben-eb" title="Reviewed Pull Requests">👀</a></td>
    <td align="center"><a href="https://github.com/Jameskmonger"><img src="https://avatars.githubusercontent.com/u/2037007?v=3?s=100" width="100px;" alt=""/><br /><sub><b>James Monger</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=Jameskmonger" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/chrissimpkins"><img src="https://avatars.githubusercontent.com/u/4249591?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Chris Simpkins</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=chrissimpkins" title="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors/pulls?q=is%3Apr+reviewed-by%3Achrissimpkins" title="Reviewed Pull Requests">👀</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://github.com/fhemberger"><img src="https://avatars.githubusercontent.com/u/153481?v=3?s=100" width="100px;" alt=""/><br /><sub><b>F. Hemberger</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=fhemberger" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/frigginglorious"><img src="https://avatars.githubusercontent.com/u/3982200?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Daniel Kraft</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=frigginglorious" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/mbad0la"><img src="https://avatars.githubusercontent.com/u/8503331?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Mayank Badola</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=mbad0la" title="Documentation">📖</a> <a href="#tool-mbad0la" title="Tools">🔧</a></td>
    <td align="center"><a href="https://www.marcobiedermann.com"><img src="https://avatars.githubusercontent.com/u/5244986?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Marco Biedermann</b></sub></a><br /><a href="#design-marcobiedermann" title="Design">🎨</a></td>
    <td align="center"><a href="https://github.com/itaisteinherz"><img src="https://avatars.githubusercontent.com/u/22768990?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Itai Steinherz</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=itaisteinherz" title="Documentation">📖</a></td>
    <td align="center"><a href="http://nodescription.net"><img src="https://avatars1.githubusercontent.com/u/305339?v=3?s=100" width="100px;" alt=""/><br /><sub><b>Patrick Connolly</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=patcon" title="Documentation">📖</a></td>
    <td align="center"><a href="http://nikolalsvk.github.io/"><img src="https://avatars2.githubusercontent.com/u/3028124?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Nikola Đuza</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=nikolalsvk" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://dem.be"><img src="https://avatars2.githubusercontent.com/u/5346497?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Demian Dekoninck</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=DemianD" title="Code">💻</a></td>
    <td align="center"><a href="https://github.com/mpeyper"><img src="https://avatars0.githubusercontent.com/u/23029903?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Michael Peyper</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=mpeyper" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/The24thDS"><img src="https://avatars0.githubusercontent.com/u/26633429?v=4?s=100" width="100px;" alt=""/><br /><sub><b>David Sima</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=The24thDS" title="Documentation">📖</a> <a href="#translation-The24thDS" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/all-contributors/all-contributors-bot"><img src="https://avatars3.githubusercontent.com/u/46843839?v=4?s=100" width="100px;" alt=""/><br /><sub><b>allcontributors[bot]</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=allcontributors" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/apps/greenkeeper"><img src="https://avatars3.githubusercontent.com/in/505?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Greenkeeper[bot]</b></sub></a><br /><a href="#infra-Greenkeeper[bot]" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a></td>
    <td align="center"><a href="https://github.com/maryampaz"><img src="https://avatars1.githubusercontent.com/u/30090413?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Maryam Pazirandeh</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=maryampaz" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/CassVenere"><img src="https://avatars1.githubusercontent.com/u/47280556?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Cassandra Venere</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=CassVenere" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://www.codimiracle.com"><img src="https://avatars2.githubusercontent.com/u/21952540?v=4?s=100" width="100px;" alt=""/><br /><sub><b>codimiracle</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=codimiracle" title="Documentation">📖</a></td>
    <td align="center"><a href="https://twitter.com/dance2die"><img src="https://avatars1.githubusercontent.com/u/8465237?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Sung Kim</b></sub></a><br /><a href="#translation-dance2die" title="Translation">🌍</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=dance2die" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/tphbrok"><img src="https://avatars0.githubusercontent.com/u/11331876?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Thomas Brok</b></sub></a><br /><a href="#translation-tphbrok" title="Translation">🌍</a></td>
    <td align="center"><a href="https://robert.theguys.sh"><img src="https://avatars0.githubusercontent.com/u/35585466?v=4?s=100" width="100px;" alt=""/><br /><sub><b>robertgrzonka</b></sub></a><br /><a href="#translation-robertgrzonka" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/askareija"><img src="https://avatars3.githubusercontent.com/u/21377617?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Megumi Aliya</b></sub></a><br /><a href="#translation-askareija" title="Translation">🌍</a></td>
    <td align="center"><a href="https://yuhang.live"><img src="https://avatars3.githubusercontent.com/u/13712499?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Yule</b></sub></a><br /><a href="#translation-YuleYu" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/s-pace"><img src="https://avatars2.githubusercontent.com/u/32097720?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Sylvain Pace</b></sub></a><br /><a href="#plugin-s-pace" title="Plugin/utility libraries">🔌</a></td>
  </tr>
  <tr>
    <td align="center"><a href="http://www.peterhuerlimann.li"><img src="https://avatars2.githubusercontent.com/u/18031711?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Peter Hürlimann</b></sub></a><br /><a href="#translation-peterhuerlimann" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/thiagodp"><img src="https://avatars3.githubusercontent.com/u/2997844?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Thiago Delgado Pinto</b></sub></a><br /><a href="#translation-thiagodp" title="Translation">🌍</a></td>
    <td align="center"><a href="https://rogeriopradoj.com"><img src="https://avatars3.githubusercontent.com/u/443391?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Rogerio Prado de Jesus</b></sub></a><br /><a href="#translation-rogeriopradoj" title="Translation">🌍</a></td>
    <td align="center"><a href="https://www.linkedin.com/in/turbolego/"><img src="https://avatars3.githubusercontent.com/u/2650749?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Tobias Andersen</b></sub></a><br /><a href="#translation-turbolego" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/danielbronder"><img src="https://avatars2.githubusercontent.com/u/9819957?v=4?s=100" width="100px;" alt=""/><br /><sub><b>danielbronder</b></sub></a><br /><a href="#translation-danielbronder" title="Translation">🌍</a></td>
    <td align="center"><a href="http://lattes.cnpq.br/4287615973321905"><img src="https://avatars3.githubusercontent.com/u/28638133?v=4?s=100" width="100px;" alt=""/><br /><sub><b>João Pedro Raskopf</b></sub></a><br /><a href="#translation-jprask" title="Translation">🌍</a></td>
    <td align="center"><a href="http://edwinvargas.com.ve"><img src="https://avatars0.githubusercontent.com/u/9091905?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Edwin Vargas</b></sub></a><br /><a href="#translation-edwinvrgs" title="Translation">🌍</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://kanout.com"><img src="https://avatars0.githubusercontent.com/u/6838659?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Faisal KANOUT</b></sub></a><br /><a href="#translation-fkanout" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/johnmurphy01"><img src="https://avatars2.githubusercontent.com/u/2939548?v=4?s=100" width="100px;" alt=""/><br /><sub><b>John Murphy</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=johnmurphy01" title="Documentation">📖</a></td>
    <td align="center"><a href="https://whitakerlab.github.io"><img src="https://avatars1.githubusercontent.com/u/3626306?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Kirstie Whitaker</b></sub></a><br /><a href="#ideas-KirstieJane" title="Ideas, Planning, & Feedback">🤔</a></td>
    <td align="center"><a href="https://github.com/sirpeas"><img src="https://avatars3.githubusercontent.com/u/4818642?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Patryk Peas</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=sirpeas" title="Documentation">📖</a> <a href="#translation-sirpeas" title="Translation">🌍</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=sirpeas" title="Code">💻</a></td>
    <td align="center"><a href="https://almostover.ru"><img src="https://avatars2.githubusercontent.com/u/16944225?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ivan.Nginx</b></sub></a><br /><a href="#translation-ivan-nginx" title="Translation">🌍</a></td>
    <td align="center"><a href="http://levy.work"><img src="https://avatars3.githubusercontent.com/u/9384365?v=4?s=100" width="100px;" alt=""/><br /><sub><b>levy</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=levy9527" title="Documentation">📖</a></td>
    <td align="center"><a href="https://piksel.se"><img src="https://avatars2.githubusercontent.com/u/807383?v=4?s=100" width="100px;" alt=""/><br /><sub><b>nils måsén</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=piksel" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="http://uraway.hatenablog.com/"><img src="https://avatars3.githubusercontent.com/u/15242484?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Masato Urai (@uraway_)</b></sub></a><br /><a href="#translation-uraway" title="Translation">🌍</a></td>
    <td align="center"><a href="https://kylemh.com"><img src="https://avatars1.githubusercontent.com/u/9523719?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Kyle Holmberg</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=kylemh" title="Documentation">📖</a></td>
    <td align="center"><a href="http://www.arcticbit.se"><img src="https://avatars0.githubusercontent.com/u/1596025?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Simon Aronsson</b></sub></a><br /><a href="#translation-simskij" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/BayCem"><img src="https://avatars0.githubusercontent.com/u/21110691?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Cem</b></sub></a><br /><a href="#translation-BayCem" title="Translation">🌍</a></td>
    <td align="center"><a href="https://github.com/fennecdjay"><img src="https://avatars0.githubusercontent.com/u/4943921?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jérémie Astor</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=fennecdjay" title="Documentation">📖</a></td>
    <td align="center"><a href="https://rachelcarmena.github.io"><img src="https://avatars0.githubusercontent.com/u/22792183?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Rachel M. Carmena</b></sub></a><br /><a href="#translation-rachelcarmena" title="Translation">🌍</a></td>
    <td align="center"><a href="https://sno2wman.dev/"><img src="https://avatars3.githubusercontent.com/u/15155608?v=4?s=100" width="100px;" alt=""/><br /><sub><b>SnO₂WMaN</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=SnO2WMaN" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://github.com/cesar-richard"><img src="https://avatars0.githubusercontent.com/u/5199868?v=4?s=100" width="100px;" alt=""/><br /><sub><b>César Richard</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=cesar-richard" title="Documentation">📖</a> <a href="#userTesting-cesar-richard" title="User Testing">📓</a></td>
    <td align="center"><a href="https://github.com/kharaone"><img src="https://avatars1.githubusercontent.com/u/6599271?v=4?s=100" width="100px;" alt=""/><br /><sub><b>kharaone</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=kharaone" title="Documentation">📖</a></td>
    <td align="center"><a href="http://thefactsbook.com"><img src="https://avatars0.githubusercontent.com/u/24487349?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Mudassar Ali</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=Mudassar045" title="Documentation">📖</a></td>
    <td align="center"><a href="https://www.andrewmason.me/"><img src="https://avatars1.githubusercontent.com/u/18423853?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Andrew Mason</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=andrewmcodes" title="Documentation">📖</a></td>
    <td align="center"><a href="https://maurom.dev"><img src="https://avatars1.githubusercontent.com/u/22800592?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Mauro M.</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=MM-coder" title="Documentation">📖</a></td>
    <td align="center"><a href="https://phacks.dev/"><img src="https://avatars1.githubusercontent.com/u/2587348?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Nicolas Goutay</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=phacks" title="Documentation">📖</a></td>
    <td align="center"><a href="http://phor.net"><img src="https://avatars0.githubusercontent.com/u/382183?v=4?s=100" width="100px;" alt=""/><br /><sub><b>William Entriken</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=fulldecent" title="Code">💻</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://linkedin.com/in/kytwb"><img src="https://avatars0.githubusercontent.com/u/412895?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Amine</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=kytwb" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/smoia"><img src="https://avatars3.githubusercontent.com/u/35300580?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Stefano Moia</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=smoia" title="Documentation">📖</a></td>
    <td align="center"><a href="http://adamtuttle.codes"><img src="https://avatars2.githubusercontent.com/u/46990?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Adam Tuttle</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=atuttle" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/NotMoni"><img src="https://avatars2.githubusercontent.com/u/40552237?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Moni</b></sub></a><br /><a href="#infra-NotMoni" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a></td>
    <td align="center"><a href="https://github.com/ilai-deutel"><img src="https://avatars0.githubusercontent.com/u/10098207?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ilaï Deutel</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=ilai-deutel" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/nhnb"><img src="https://avatars1.githubusercontent.com/u/364184?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Hendrik Brummermann</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=nhnb" title="Documentation">📖</a></td>
    <td align="center"><a href="https://www.weareaccess.co.uk/"><img src="https://avatars0.githubusercontent.com/u/4610533?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Sang Lostrie</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/issues?q=author%3Abaikho" title="Bug reports">🐛</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://github.com/xinbenlv"><img src="https://avatars2.githubusercontent.com/u/640325?v=4?s=100" width="100px;" alt=""/><br /><sub><b>xinbenlv</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=xinbenlv" title="Documentation">📖</a></td>
    <td align="center"><a href="https://paulovich.net"><img src="https://avatars3.githubusercontent.com/u/7133698?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ivan Paulovich</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=ivanpaulovich" title="Documentation">📖</a></td>
    <td align="center"><a href="https://www.jakewiesler.com"><img src="https://avatars1.githubusercontent.com/u/12075916?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jake Wiesler</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=jakewies" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/MicaelJarniac"><img src="https://avatars0.githubusercontent.com/u/19514231?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Micael Jarniac</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/issues?q=author%3AMicaelJarniac" title="Bug reports">🐛</a></td>
    <td align="center"><a href="https://www.linkedin.com/in/mloning/"><img src="https://avatars3.githubusercontent.com/u/21020482?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Markus Löning</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=mloning" title="Documentation">📖</a></td>
    <td align="center"><a href="https://austinhuang.me"><img src="https://avatars1.githubusercontent.com/u/16656689?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Austin Huang</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=austinhuang0131" title="Documentation">📖</a></td>
    <td align="center"><a href="http://www.nils-andresen.de"><img src="https://avatars3.githubusercontent.com/u/349188?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Nils Andresen</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=nils-a" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://github.com/LaChapeliere"><img src="https://avatars2.githubusercontent.com/u/7062546?v=4?s=100" width="100px;" alt=""/><br /><sub><b>LaChapeliere</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=LaChapeliere" title="Documentation">📖</a></td>
    <td align="center"><a href="https://dev.to/mbiesiad"><img src="https://avatars0.githubusercontent.com/u/18367606?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Michal</b></sub></a><br /><a href="#translation-mbiesiad" title="Translation">🌍</a></td>
    <td align="center"><a href="https://www.linkedin.com/in/alitorki/"><img src="https://avatars1.githubusercontent.com/u/9049092?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ali Torki</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=ali-master" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/jsoref"><img src="https://avatars0.githubusercontent.com/u/2119212?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Josh Soref</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=jsoref" title="Documentation">📖</a></td>
    <td align="center"><a href="https://www.taiizor.com"><img src="https://avatars.githubusercontent.com/u/41683699?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Taiizor</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=Taiizor" title="Code">💻</a></td>
    <td align="center"><a href="https://bandism.net/"><img src="https://avatars.githubusercontent.com/u/22633385?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ikko Ashimine</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=eltociear" title="Documentation">📖</a></td>
    <td align="center"><a href="https://twitter.com/MatthewTFoley"><img src="https://avatars.githubusercontent.com/u/3792749?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Matthew</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=mtfoley" title="Documentation">📖</a></td>
  </tr>
  <tr>
    <td align="center"><a href="https://github.com/corneliusroemer"><img src="https://avatars.githubusercontent.com/u/25161793?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Cornelius Roemer</b></sub></a><br /><a href="#design-corneliusroemer" title="Design">🎨</a></td>
    <td align="center"><a href="https://turnipguy30.github.io"><img src="https://avatars.githubusercontent.com/u/50542928?v=4?s=100" width="100px;" alt=""/><br /><sub><b>JohnnySD</b></sub></a><br /><a href="#content-TurnipGuy30" title="Content">🖋</a> <a href="https://github.com/all-contributors/all-contributors/commits?author=TurnipGuy30" title="Documentation">📖</a></td>
    <td align="center"><a href="https://github.com/mwaitzman"><img src="https://avatars.githubusercontent.com/u/51432220?v=4?s=100" width="100px;" alt=""/><br /><sub><b>mwaitzman</b></sub></a><br /><a href="https://github.com/all-contributors/all-contributors/commits?author=mwaitzman" title="Documentation">📖</a></td>
  </tr>
</table>
